README.md

```diff
@@ -1,52 +1,11 @@
 401 JS -- Class 01 Lab modular patterns
```

Comment:
Read the **Documentation** parts of the labs again. Remember you need to write some things in your README per the lab instructions.